pyicloud/services/photos.py

```diff
@@ -499,15 +499,21 @@ def __init__(self, service, master_record, asset_record):
         self._versions = None
 
     PHOTO_VERSION_LOOKUP = {
-        u"original": u"resOriginal",
+        u"full": u"resJPEGFull",
```

Comment:
A hint would be handy, but completely optional:

```suggestion
        u"full": u"resJPEGFull",  # Full image with all editions applied.
```

I appreciate the effort in finding this! I take a lot of photos and scribble notes on them, and this is a key item to be able to sync a specific album into my laptop continuously :relieved: 

How did you determine these aliases, BTW? This downloads the image with any editions I've done in the phone, but rather than download a HEIC image, it's a JPEG.

Any ideas how to get the actual HEIC file itself? This JPEG seems to have metadata stripped (e.g.: no location data. It looks like the version that iOS would share if you try to share a photo from the library).

Reply:
Oh, I found this issue which answers some of my questions: https://github.com/icloud-photos-downloader/icloud_photos_downloader/issues/249

I don't see `resOriginal` and a few others listed in `photo._asset_record`; any idea how _those_ aliases came to be?

Reply:
Thanks for the review @WhyNotHugo. I'm not sure that `resJPEGFull` always contains the edited version of the photo. 

I *think* it has more to do with whether the image is retrieved from `CPLAsset` or `CPLMaster` in the response. `CPLAsset` appears to contain the edited versions in various file sizes and formats.